### ISSUE TITLE
Winglet Textures not working

### ISSUE THREAD

**Issue by Pakistan-1** [soitanen/737-800#25]
The textures for the winglets are not working

**Comment by soitanen** [comment]
Yes, it's known bug. Currently I use new winglets model from Milo. But I don't know anything about 3D modelling and textures, so I just live with this issue.

**Comment by GabrielVenezuela** [comment]
problem solved for my soon will push the correction 👍
